docker-alpine/Dockerfile

```
@@ -0,0 +1,88 @@
+FROM alpine:latest
+
+#Otherwise mosquitto fails
+VOLUME ["/sys/fs/cgroup"]
+
+#We need curl to get root certificates
+RUN apk update
+RUN apk upgrade
+RUN apk add mosquitto
+RUN apk add curl
+RUN apk add collectd
+RUN apk add sudo
+RUN apk add openrc
+RUN apk add bash
```

```suggestion
RUN apk update \
    && apk upgrade \
    && apk --no-cache add \
    mosquitto \
    curl \
    collectd \
    sudo \
    openrc \
    bash \
```

This way it is treated as a single step and in in case it fails as you have not specified install version it will always install latest stable and use no additional space to store cache.